src/layouts/PDFViewer.astro

```diff
@@ -1,96 +1,62 @@
 ---
-import BasicMeta from '../components/BasicMeta.astro'
-import Header from '../components/Header.astro'
-import Footer from '../components/Footer.astro'
-import Global from '../components/Global.astro'
+import MainLayout from './MainCentered.astro'
 import CtaButton from '../components/CtaButton.astro'
 const { content } = Astro.props
 ---
-
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <BasicMeta title={`Vivic Research | ${content.title}`} />
-    <Global />
-</head>
-
-<body>
-    <Header />
-    <main>
-        <h1>{content.title}</h1>
-        <a class=topic href={`/work?topic=${content.topic.toLowerCase()}`}>{content.topic}</a>
-        <div id=pdf>
-            <iframe src={content.pdf}></iframe>
-        </div>
-        <CtaButton link={content.pdf}> Download </CtaButton>
-    </main>
-    <Footer />
-    <script data-goatcounter="https://vivic.goatcounter.com/count" async src="//gc.zgo.at/count.js"></script>
-</body>
-</html>
-
-
+<MainLayout title={`Vivic Research | ${content.title}`}>
+	<h1>{content.title}</h1>
+	<a
+		class="topic" 
+		href={`/work?topic=${content.topic.toLowerCase()}`}
+	>
+		{content.topic}
+	</a>
+	<div id="pdf">
+		<iframe src={content.pdf}/>
+	</div>
+	<CtaButton link={content.pdf}>Download</CtaButton>
+</MainLayout>
 
 <style>
-body {
-    color: var(--white);
-    min-height: 100vh;
-    background-image: radial-gradient(88% 140% at 120% -5%, #482979, #40256b, #38215e, #301d51, #291943, #211536, var(--dark) 80%);
-    background-size: cover;
-    background-repeat: no-repeat;
-    background-color: var(--dark);
-    margin: 0;
-}
-
-main {
-    width: 80%;
-    margin: auto;
-    margin-top: 250px;
-    font-family: var(--mainfont);
-    min-height: 100vh;
-}
-
-#pdf {
-    height: 85vh;
-    width: 85%;
-    margin: auto;
-    margin-bottom: 80px;
-}
-
-iframe {
-    height: 100%;
-    width: 100%;
-}
```

Comment:
I've just moved the `#pdf` and `iframe` statements to the bottom to better reflect the order of the page.